appender.go

```diff
@@ -141,7 +141,11 @@ func New(client *elasticsearch.Client, cfg Config) (*Appender, error) {
 	}
 	available := make(chan *BulkIndexer, cfg.MaxRequests)
 	for i := 0; i < cfg.MaxRequests; i++ {
-		available <- NewBulkIndexer(client, cfg.CompressionLevel, cfg.MaxDocumentRetries)
+		available <- NewBulkIndexer(client, BulkIndexerConfig{
```

Comment:
To avoid breaking the API in the future, wouldn't it be better to also pass the Elasticsearch Client in `BulkIndexerConfig`?

Reply:
Agreed. I did this only for consistency with `func New(client esapi.Transport, cfg Config) (*Appender, error) {`. Will update this PR after #146